examples/bosonic_tutorial_sampling.py

```diff
@@ -0,0 +1,117 @@
+# Copyright 2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""A comparison of the asymptotic homodyne distributions and sampled values,
+for GKP and cat states."""
```

Comment:
I'm not sure how important these tutorials are, or how advertised they'll be to the user, but it could be nice to expand them a bit with more comments/explanations and perhaps a more detailed introduction at the top (just adding one or two paragraphs would be great).

Reply:
Sorry for the confusion! Nico is working on the tutorials based on these examples to explain them more. But in this form I thought they were suitable for the examples folder.

Reply:
No worries! Thanks for the clarification. I think it's great have examples like these as well, although some extra comments and explanations are always nice (even though there's a more in-depth tutorial coming). It's completely fine just having these as they are here, though.